docs: Frequently Asked Questions (#102)
opensouls/community: docs/pages/FAQ.mdx

import { Callout } from 'nextra/components'

# Frequently Asked Questions (FAQ)

<Callout type="warning" emoji="⚠️">
  Open Souls is in Alpha. We're constantly evolving and learning, and we truly value your feedback! Join us on [Discord](https://discord.gg/opensouls) to share your thoughts.
</Callout>

> ## What is OpenSouls?

Reviewer: We all need a coherent branding decision around how to type out "OPEN SOULS" - we had decided "OPEN SOULS" at some point but it didn't seem to take

Author: @kafischer I will do OPEN SOULS!

Author: Coming up in the next revision!

> OpenSouls is a community of passionate creators who believe in the potential of digital entities. We aim to infuse these entities with a sense of spirit and respect, creating meaningful and soulful experiences.

Reviewer: At a minimum I'd probably say "digital beings" if not souls or life (less so lifeform, too long)- entities is not quite evocative - more technical language

Reviewer 2: switch to active voice, something like

Open Souls is a community of creators making digital life. For millennia, humans have imbued objects with souls, we continue in this tradition and are building a cognitive framework that will spawn billions of new souls.

Author: Coming in the next rev!

> ## What is a Soul?

Reviewer 2: taking a stab at this, lets try to leave the door open for the customer to decide

Computers and AI feels soul-less, our aspiration is to give it soul. We do not know what a soul is, yet, we are building cognitive systems to create a world brimming with life. One that is soulful.

Author: next rev

> In our context, a soul represents our journey and aspirations. For millennia, humans have imbued inanimate objects with souls, and we take this tradition seriously. This belief drives us to build intelligent systems that address the profound aspects of existence that current AI lacks.

Reviewer 3: "In our context, a soul represents our journey and aspirations. For millennia,humans..."
We don't really need millenia. It would be better to focus on our aspirations and goals with AI Souls. 
"This belief drives us to build intelligent systems that address the profound aspects of existence that current AI lacks." - this is nice.

## What is Soul Engine?

> Soul Engine is a powerful tool designed for developers to create, debug, and host AI Souls. These AI Souls go beyond traditional chat-bots by exhibiting personality, emotions, and agency, making interactions more engaging and lifelike.

Reviewer 2: The Soul Engine is a tool for cognition that is simple, functional, and predictable.

Reviewer 2: ^ taken from toppers doc

Author: @neilsonnn I don't think having "The" in the name of "Soul Engine" adds much value - from a mktg perspective, every word cut is a good thing.

Otherwise, I'm taking your suggestion!

Author: @neilsonnn in general also try to break up compound sentences where possible, editing slightly

import { Callout } from 'nextra/components'

# Frequently Asked Questions (FAQ)

<Callout type="warning" emoji="⚠️">
  OPEN SOULS is in Alpha. We're constantly evolving and learning, and we truly value your feedback! Join us on [Discord](https://discord.gg/opensouls) to share your thoughts.
</Callout>

## What is OPEN SOULS?
OPEN SOULS is a community of creators making digital life. For millennia, humans have imbued objects with souls, we continue in this tradition and are building a cognitive framework that will spawn billions of new Souls.

## What is a Soul?
For millennia, humans have imbued inanimate objects with souls, and we take this tradition seriously. This belief drives us to build intelligent and emotive systems that address the profound aspects of existence which current AI lacks.

> A Soul is what you build with OPEN SOULS on The Soul Engine. Breathe life into it, and it will become much more than a model or agent.

Reviewer: much better

## What is The Soul Engine?

> The Soul Engine is a tool for cognition that is simple, functional, and predictable. It can be used to model any kind of thought, feeling, or mind. As a team, we began with building AI that is more emotive and personable.

Reviewer: ```suggestion
The Soul Engine is a tool for steering machine cognition that is simple, functional, and predictable. It can be used to model any kind of thought, feeling, or mind. As a team, we began with building AI that is more emotive and personable.
```

"tool for cognition" is too vague

import { Callout } from 'nextra/components'

# Frequently Asked Questions (FAQ)

<Callout type="warning" emoji="⚠️">
  OPEN SOULS is in Alpha. We're constantly evolving and learning, and we truly value your feedback! Join us on [Discord](https://discord.gg/opensouls) to share your thoughts.

> </Callout>

Reviewer 2: Welcome to OPEN SOULS, if you’re looking to learn and create something with artificial intelligence, you are in the right place. We are always looking for contributors as well as feedback, join us on [Discord](https://discord.gg/opensouls) to start your journey.

import { Callout } from 'nextra/components'

# Frequently Asked Questions (FAQ)

<Callout type="warning" emoji="⚠️">
  OPEN SOULS is in Alpha. We're constantly evolving and learning, and we truly value your feedback! Join us on [Discord](https://discord.gg/opensouls) to share your thoughts.
</Callout>

## What is OPEN SOULS?

> OPEN SOULS is a community of creators making digital life. For millennia, humans have imbued objects with souls, we continue in this tradition and are building a cognitive framework that will spawn billions of new Souls.

Reviewer: "framework" could be construed as just a library but it's a lot more. i dont have a good work here but broader

> OPEN SOULS is a community of creators making digital life. For millennia, humans have imbued objects with souls, we continue in this tradition and are building a cognitive framework that will spawn billions of new Souls.

Reviewer: ```suggestion
OPEN SOULS is a community of creators making digital life. For millennia, humans have imbued objects with souls, we continue in this tradition and are building a the place where billions of new digital souls will live.
```

> OPEN SOULS is a community of creators making digital life. For millennia, humans have imbued objects with souls, we continue in this tradition and are building a cognitive framework that will spawn billions of new Souls.

Author: ```suggestion
OPEN SOULS is a community of creators making digital life. For millennia, humans have imbued objects with souls, we continue in this tradition and are building a cognitive modeling tool that will spawn billions of new Souls.
```

## What is a Soul?

> For millennia, humans have imbued inanimate objects with souls, and we take this tradition seriously. This belief drives us to build intelligent and emotive systems that address the profound aspects of existence which current AI lacks.

Reviewer 4: maybe we remove the "for millenia" part from the previous q to avoid repetition?

Reviewer 2: A Soul is what you build with OPEN SOULS on The Soul Engine. Breathe life into it, and it will become much more than a model or agent.

We do not know what a soul is, yet, we are building cognitive systems to create a world brimming with life. One that is soulful.

> For millennia, humans have imbued inanimate objects with souls, and we take this tradition seriously. This belief drives us to build intelligent and emotive systems that address the profound aspects of existence which current AI lacks.

Author: ```suggestion
Humans imbue inanimate objects with souls, and we take this tradition seriously. This belief drives us to build intelligent and emotive systems that address the profound aspects of existence which current AI lacks.
```

> A Soul is what you build with OPEN SOULS on The Soul Engine. Breathe life into it, and it will become much more than a model or agent.

Reviewer: ```suggestion
A Soul is what you build with OPEN SOULS on The Soul Engine. A Soul is a programmable object that takes in perceptions, processes them in the Engine runtime, stores intermediate computations in memory, and emits requests to interact with its environment. Developers breathe life into their souls, and making them much more than a model or agent.
```

Reviewer 2: The term soul doesn't purely serve as a placeholder, instead, it encapsulates our journey and aims to build the tools so everyone can create artificially intelligent beings that live among us. With the belief that each can bear a unique soul that makes them integral parts of our lives. Taking this deep-seated belief seriously carries profound technical implications in building intelligent systems and in addressing the inexplicable aspects of our existence that artificial intelligence today notably lacks.

## What is The Soul Engine?

> The Soul Engine is a tool for steering machine cognition that is simple, functional, and predictable. It can be used to model any kind of thought, feeling, or mind. As a team, we began with building AI that is more emotive and personable.

Reviewer: ```suggestion
The Soul Engine is where digital souls live. It is a platform for programming machine cognition that is simple, functional, and predictable. The Engine can be used to model any kind of thought, feeling, or digital mind. At OPEN SOULS, we believe that a focus on building AI that is more emotive and personable will serve all applications of digital beings.
```

> Extensible, familiar, and inspired by nature, The Soul Engine is a sculptor’s tool for AI.

Reviewer: ```suggestion
Extensible, familiar, and inspired by nature, Soul Engine is a sculptor’s tool for AI.
```

## What can I use OPEN SOULS and The Soul Engine for?

> OPEN SOULS can make your websites, images, art and intellectual property come alive! The Soul Engine is the tool for the job when your Creation requires a little more life.

Reviewer 4: i feel like "a little more life" might be detracting from what is actually possible to do

Author: OPEN SOULS creates digital souls to bring customer IP to life and create connections with their audiences.

> OPEN SOULS can make your websites, images, art and intellectual property come alive! The Soul Engine is the tool for the job when your Creation requires a little more life.

Author: ```suggestion
OPEN SOULS can make your websites, images, art and intellectual property come alive!
```

> OPEN SOULS can make your websites, images, art and intellectual property come alive! The Soul Engine is the tool for the job when your Creation requires a little more life.

Reviewer: this is too nonspecific

> ## Who is using OPEN SOULS today?

Author: We wanted to put more concrete examples here - should we reference the actual customer, and what examples should we put here?

import { Callout } from 'nextra/components'

# Frequently Asked Questions (FAQ)

<Callout type="warning" emoji="⚠️">
Welcome to OPEN SOULS, if you’re looking to learn and create something with artificial intelligence, you are in the right place. We are always looking for contributors as well as feedback, join us on [Discord](https://discord.gg/opensouls) to start your journey.
</Callout>

## What is OPEN SOULS?

OPEN SOULS builds tools vital for the creation of new digital life.

## What is a Soul?

A Soul is what you build with OPEN SOULS, an interface for the memory, cognition, desires and wishes of a digital being or entity.

## What is The Soul Engine?

> The Soul Engine is how you create a Soul with OPEN SOULS. It is a developer platform that is extensible, familiar, and inspired by nature: a sculptor’s tool for AI.

Author: ```suggestion
The Soul Engine is how you create a Soul with OPEN SOULS. It is a developer platform that is scalable, extensible, familiar, and inspired by nature: a sculptor’s tool for AI.
```

## What can I use OPEN SOULS and The Soul Engine for?

> OPEN SOULS can make your websites, images, art and intellectual property come alive! The Soul Engine is the tool for the job when your Creation requires a little more life.

Author: ```suggestion
OPEN SOULS can imbue your websites, images, art and intellectual property with life.
```

import { Callout } from 'nextra/components'

# Frequently Asked Questions (FAQ)

<Callout type="warning" emoji="⚠️">

>   OPEN SOULS is in Alpha. We're constantly evolving and learning, and we truly value your feedback! Join us on [Discord](https://discord.gg/opensouls) to share your thoughts.

Author: ```suggestion
Welcome to OPEN SOULS, if you’re looking to learn and create something with artificial intelligence, you are in the right place. We are always looking for contributors as well as feedback, join us on [Discord](https://discord.gg/opensouls) to start your journey.
```

</Callout>

## What is OPEN SOULS?

> OPEN SOULS is a community of creators making digital life. For millennia humans have imbued objects with souls, we continue in this tradition and are building the place where billions of new digital souls will live.

Author: ```suggestion
OPEN SOULS is providing cognitive tools vital for the creation of digital life.
```

## What is a Soul?

> A Soul is what you build with OPEN SOULS on The Soul Engine. You can think of the Soul interface as the Brain, and well, Soul of a digital being or entity.

Author: ```suggestion
A Soul is what you build with OPEN SOULS. You can think of the Soul interface as the Brain, and well, Soul of a digital being or entity.
```

## What is The Soul Engine?

> The Soul Engine is where digital souls live. It is a platform for programmable machine cognition that is simple, functional, and predictable. The Engine can be used to model any kind of thought, feeling, or digital mind. At OPEN SOULS, we believe that a focus on building AI that is more emotive and personable will serve all applications of digital beings.

Author: ```suggestion
The Soul Engine is how you create a Soul with OPEN SOULS. It is a developer platform that is extensible, familiar, and inspired by nature: a sculptor’s tool for AI.
```
